packages/tx/package.json

```diff
@@ -46,7 +49,7 @@
     "prepublishOnly": "../../config/cli/prepublish.sh",
     "test": "npm run test:node && npm run test:browser",
     "test:browser": "npx vitest run --config=./vitest.config.browser.mts",
-    "test:node": "npx vitest run",
+    "test:node": "NODE_OPTIONS='--conditions=typescript' vitest run",
```

Comment:
This looks like an invalid command and I think we want to keep all packages the same (using old behavior?)

Reply:
Good catch, thanks!